package.json

```diff
@@ -3,11 +3,12 @@
   "version": "1.0.0",
   "description": "**OpenPoll** is an open-source Slack-integrated app that lets your team quickly create and run polls using a `/poll` command. Inspired by [Simple Poll](https://www.simplepoll.rocks/), OpenPoll is built in **TypeScript**, runs on **Firebase Cloud Functions**, and is fully customizable for self-hosting or development learning.",
   "main": "index.js",
-  "dependencies": {
-   
-  },
   "devDependencies": {
-    "husky": "^9.1.7"
+    "@firebase/rules-unit-testing": "^5.0.0",
+    "axios": "^1.10.0",
```

Comment:
I think we should not have this in the root package.json, if we have something it should live in the functions/package.json , please remove it from here. Unless it was already here. Also I think these packages are from testing, which we didn't implement in the end, so don't add them either there.